esx_service/cli/vmdkops_admin.py

```diff
@@ -248,13 +253,14 @@ def commands():
                                 }
                             }
                         },
+
                         'ls': {
                             'help': "list VMs in a tenant",
                             'func': tenant_vm_ls,
                             'args': {
                                 '--name': {
                                     'help': "Tenant to list the VMs for",
-                                    'required': False
+                                    'required': True
```

Comment:
Not giving the name filter should allow listing all VMs in all tenants similar to what "ls" would do without "-c" or "--tenant" options.